scout/server/blueprints/cases/controllers.py

```diff
@@ -765,9 +765,8 @@ def mt_excel_files(store, case_obj, temp_excel_dir):
             for col, field in enumerate(line):  # each field in line becomes a cell
                 Report_Sheet.write(row, col, field)
 
-        if bool(
-            coverage_stats
-        ):  # it's None if app is not connected to Chanjo or {} if samples are not in Chanjo db
+        if coverage_stats and coverage_stats.get(sample_id):
```

Comment:
Is there something particular reason to test also `bool(coverage_stats.get(sample_id))` (e.g. it in turn returning None or {})? If so add to comment since it has two of the four tested conditions already! 😊 
Alternatively consider testing 
```suggestion
        if coverage_stats and sample_id in coverage_stats:
```

Reply:
coverage_stats can be None, {} or a dictionary with keys. 
I can reintroduce the comment!

Reply:
No worries, you only moved it - and the sample_id in coverage_stats says it all already - thanks!